profiles/appuio.json

```diff
@@ -0,0 +1,221 @@
+{
+  "name": "APPUiO - Swiss Container Platform",
+  "revision": "2016-10-05",
+  "url": "https://appuio.ch",
+  "vendor_verified": "2016-10-05",
+  "status": "production",
+  "status_since": "2016-10-05",
+  "type": "Generic",
+  "hosting": {
+    "public": true,
+    "private": true
```

Comment:
You can also add `vps: true` as you offer to host private single tenant instances for your customers?!